kie-server-parent/kie-server-services/kie-server-services-optaplanner/src/main/java/org/kie/server/services/optaplanner/SolverServiceBase.java

```diff
@@ -515,4 +516,41 @@ private void terminateSolverEarly(SolverInstanceContext sic) {
         }
         sic.getSolver().terminateEarly();
     }
+    
+    protected boolean boostrapContinuousPlanning(String containerId, List<ContinuousPlanning> plannings) {
+        for (ContinuousPlanning planning : plannings) {
+            if (planning.accept(containerId)) {                
+                logger.info("ContinuousPlanning {} is going to be used for container {}", planning, containerId);
+                
+                String solverId = planning.getSolverId();
+                SolverInstance instance = new SolverInstance();
+                instance.setSolverConfigFile(solverId + "-solver.xml");
```

Comment:
I do believe we need a new method String getSolverConfigPath() in ContinousPlanning interface to return a relative path to a config file. This approach imposes certain naming format, which should be avoided. The Solver API that we have ATM allows setting the solver config path separately from the solverId used.